envoy/network/filter.h

```diff
@@ -465,6 +466,15 @@ class QuicListenerFilter {
    */
   virtual FilterStatus onPeerAddressChanged(const quic::QuicSocketAddress& new_address,
                                             Connection& connection) PURE;
+
+  /**
+   * Called when the QUIC server session processes its first packet.
+   * @param packet the received packet.
+   * @return status used by the filter manager to manage further filter iteration.
+   */
+  virtual FilterStatus onFirstPacketReceived(const quic::QuicReceivedPacket&) {
+    return FilterStatus::Continue;
```

Comment:
Can you add a TODO to make this interface pure virtual once imported?

Reply:
Done.

Reply:
We have a few internally, but we can make this PURE and update our implementations instead.